pkgs/by-name/sd/SDL_audiolib/package.nix

```diff
@@ -49,6 +68,9 @@ stdenv.mkDerivation (finalAttrs: {
     (lib.cmakeBool "USE_DEC_XMP" false)
     (lib.cmakeBool "USE_RESAMP_SOXR" false)
     (lib.cmakeBool "USE_RESAMP_SRC" false)
+  ]
+  ++ lib.optionals stdenv.isDarwin [
```

Comment:
We should do this unconditionally, unvendoring deps is a good thing for all platforms.